v2/protegrity-data-tokenization/src/main/java/com/google/cloud/teleport/v2/transforms/io/GcsIO.java

```diff
@@ -61,16 +63,27 @@
     /** Necessary {@link PipelineOptions} options for Pipelines that operate with JSON/CSV data in GCS. */
     public interface GcsPipelineOptions extends PipelineOptions {
         @Description("GCS filepattern for files in bucket to read data from")
-        String getInputGcsFilePattern();
+        String getInputGcsPath();
+
+        void setInputGcsPath(String inputGcsPath);
+
+        @Description("GCS filepattern for files in bucket to write data to")
+        String getOutputGcsPath();
```

Comment:
`outputGcsPath` option is totally okay, just, please replace the word `filepattern` in Description with `file path`.